src/main/java/com/devonfw/quarkus/productmanagement/service/v1/model/NewProductDto.java

```diff
@@ -0,0 +1,23 @@
+package com.devonfw.quarkus.productmanagement.service.v1.model;
+
+import java.math.BigDecimal;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class NewProductDto {
+
+  @Schema(nullable = false, description = "Product title", minLength = 3, maxLength = 500)
+  private String title;
+
+  @Schema(description = "Product description", minLength = 3, maxLength = 500)
+  private String description;
+
+  @Schema(description = "Product price", minLength = 1)
```

Comment:
Remove the minLength property, as this is only for strings (see [here](https://openliberty.io/docs/21.0.0.9/reference/javadoc/microprofile-4.0-javadoc.html#class=org/eclipse/microprofile/openapi/annotations/media/Schema.html#minLength()&package=allclasses-frame.html))